### ISSUE TITLE
ANSIBLE0013 and bash buildins

### ISSUE THREAD

**Issue by keachi** [ansible/ansible-lint#206]
# Issue Type
- Bug report
# Ansible and Ansible Lint details

```
ansible 2.1.1.0
ansible-lint 3.3.0
```
- ansible installation method: pip
- ansible-lint installation method: pip
# Desired Behaviour

The ansible module shell is also needed for using commands from bash buildin, e.g. ulimit.
An example task could look like:

```
- name: foo bar
  shell: ulimit -n
```

doesn't work with ansible module command. But ANSIBLE0013 reject that that task.
# Actual Behaviour (Bug report only)

ANSIBLE0013 should also include bash buildins, like ulimit. Possible list of bash buildins is `compgen -b`.
Not on every system there is an binary ulimit. There are systems which has this binary, but other systems (like a minimal centos 7 oder debian 8) doesn't have it and bash will handle the call with the buildin stuff.

**Comment by ssbarnea** [comment]
I think that adding `warn: False` should do the trick in this case. Maybe it would worth to mention this as part of the failure message.

**Comment by tdmalone** [comment]
`warn: False` doesn't appear to work for me here, but `tags: skip_ansible_lint` does... (the alternative method, according to the [README](https://github.com/willthames/ansible-lint#false-positives).

**Comment by ssbarnea** [comment]
Obsolete.
